tests/inference/expressions_test.py

```diff
@@ -1,34 +1,30 @@
-a = 1 + 2 / 3
```

Comment:
Yes I accidentally removed an addition axiom while resolving the conflicts 😅 Works fine now.